### PR TITLE
RHEL-90: Remove deprecated `crashkernel=auto` option

### LINKED ISSUE #1819: [RHEL-9] Remove deprecated option `crashkernel=auto` from all image definitions
The RHEL-only patch to support kernel option `crashkernel=auto` has been dropped in RHEL-9. For this reason none of the RHEL-9 images should use it for kernel command line.

The functionality has been reimplemented by introducing a new file `/lib/modules/$kernel_ver/crashkernel.default` installed by the `kernel-core` package, which content should be appended to the kernel command line instead.

More information in https://bugzilla.redhat.com/show_bug.cgi?id=2006692

### PR DESCRIPTION
**Pull request by thozza**
Setting of the `crashkernel` option to the appropriate value is now done
by the `kexec-tools` package when installed and when any new kernel is
installed.

Regenerate relevant image test cases.

Fix #1819
Fix [rhbz#2006692](https://bugzilla.redhat.com/show_bug.cgi?id=2006692)

Signed-off-by: Tomas Hozza <thozza@redhat.com>

This pull request includes:

- [ ] adequate testing for the new functionality or fixed issue
- [ ] adequate documentation informing people about the change such as
  - [ ] submit a PR for the [guides](https://github.com/osbuild/guides) repository if this PR changed any behavior described there: https://www.osbuild.org/guides/

### CODE REVIEW

**Review by gicmo — APPROVED** [review]
Thanks so much!

File: test/data/manifests/rhel_90-aarch64-ami-boot.json [hunk 1]
```diff
@@ -9957,6 +9957,7 @@
         "/etc/pam.d/smartcard-auth": "....L....",
         "/etc/pam.d/system-auth": "....L....",
         "/etc/rhsm/rhsm.conf": "..5....T.",
+        "/etc/ssh/sshd_config": "S.5....T.",
```
Comment (achilleas-k) [inline test/data/manifests/rhel_90-aarch64-ami-boot.json:9960]: _Someone_ didn't update the test cases properly :laughing:

**Review by achilleas-k — APPROVED** [review]
LGTM

File: test/data/manifests/rhel_90-aarch64-ami-boot.json (hunk 1 above)
Comment (gicmo) [reply test/data/manifests/rhel_90-aarch64-ami-boot.json:9960]: Totally me. I want a bot for this work. (-:

Comment (thozza) [reply test/data/manifests/rhel_90-aarch64-ami-boot.json:9960]: 🤖  Tomas at your service 🤣

Comment (gicmo) [reply test/data/manifests/rhel_90-aarch64-ami-boot.json:9960]: Thanks Tomas! ;) I. All seriousness, this should be done by a bot instead :)

Comment (thozza) [reply test/data/manifests/rhel_90-aarch64-ami-boot.json:9960]: I know, I'm joking 😜

**Pull request merged**